PhysicsTools/JetMCAlgos/plugins/HadronAndPartonSelector.cc

```diff
@@ -254,7 +254,7 @@ HadronAndPartonSelector::produce(edm::Event& iEvent, const edm::EventSetup& iSet
      partonSelector_->run(particles,partons);
      for(reco::GenParticleCollection::const_iterator it = particles->begin(); it != particles->end(); ++it)
      {
-       if( !(it->status()==3 || (( partonMode_=="Pythia8" ) && (it->status()==23)))) continue;
+      // if( !(it->status()==3 || (( partonMode_=="Pythia8" ) && (it->status()==23)))) continue;
```

Comment:
Hello @monttj ,
These changes were requested by the QGL Group, i.e. Paolo Azzuri. Unfortunately, I cannot find him on GitHub to comment on this. 

You are right, the  physicspartons will contain the whole chain. I will provide RelVal comparisons as soon as possible.

Kind regards,
Andrej